services/storage/src/simcore_service_storage/api/rest/_files.py

```diff
@@ -284,8 +284,10 @@ async def complete_upload_file(
         user_id=query_params.user_id,
         product_name=_UNDEFINED_PRODUCT_NAME_FOR_WORKER_TASKS,  # NOTE: I would need to change the API here
     )
-    task_uuid = await celery_client.send_task(
-        remote_complete_upload_file.__name__,
+    task_uuid = await celery_client.submit_task(
+        TaskMetadata(
```

Comment:
Just FYI ,please read these defiition (and the links) of resources and the difference it makes between `name` vs `id`
https://github.com/itisfoundation/osparc-simcore/blob/020e2dfc353ebb4c94eceba3a91c762703b2ec0f/services/api-server/src/simcore_service_api_server/models/api_resources.py#L8-L27